nima/webserver/webserver/src/main/java/io/helidon/nima/webserver/http1/Http1Connection.java

```diff
@@ -54,6 +55,9 @@
 public class Http1Connection implements ServerConnection, InterruptableTask<Void> {
     private static final System.Logger LOGGER = System.getLogger(Http1Connection.class.getName());
 
+    private static final byte[] UNSUPPORTED_EXPECT_417 =
+            "HTTP/1.1 417 Unsupported-Expect\r\n\r\n".getBytes(StandardCharsets.UTF_8);
+    static final byte[] CONTINUE_100 = "HTTP/1.1 100 Continue\r\n\r\n".getBytes(StandardCharsets.UTF_8);
```

Comment:
Please add new lines between fields of the same type (not required, but nicer to read).
Also private should be after package local.
So here:
```
static final .. CONTINUE_100

private static final ... UNSUPPORTED...

private final ConnectionContext
```

Reply:
fixed